sdk/web-pubsub/web-pubsub-express/README.md

```diff
@@ -0,0 +1,118 @@
+# Azure Web PubSub CloudEvents handlers for Express
+
+[Azure Web PubSub Service](https://aka.ms/awps/doc) is a service that enables you to build real-time messaging web applications using WebSockets and the publish-subscribe pattern. Any platform supporting WebSocket APIs can connect to the service easily, e.g. web pages, mobile applications, edge devices, etc. The service manages the WebSocket connections for you and allows up to 100K \*concurrent connections. It provides powerful APIs for you to manage these clients and deliver real-time messages.
+
+Any scenario that requires real-time publish-subscribe messaging between server and clients or among clients, can use Azure Web PubSub service. Traditional real-time features that often require polling from server or submitting HTTP requests, can also use Azure Web PubSub service.
+
+Use the express library to:
+
+- Add Web PubSub CloudEvents middleware to handle incoming client events
+  - Handle abuse validation requests
+  - Handle client events requests
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/web-pubsub/web-pubsub-express) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/web-pubsub-express) |
+[API reference documentation](https://aka.ms/awps/sdk/js) |
+[Product documentation](https://aka.ms/awps/doc) |
+[Samples][samples_ref]
+
+## Getting started
+
+### Currently supported environments
+
+- [Node.js](https://nodejs.org/) version 8.x.x or higher
+- [Express](https://expressjs.com/) version 4.x.x or higher
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- An existing Azure Web PubSub endpoint.
+
+### 1. Install the `@azure/web-pubsub-express` package
+
+```bash
+npm install @azure/web-pubsub-express
+```
+
+### 2. Create a WebPubSubEventHandler
+
+```js
+import express from "express";
```

Comment:
now we've mixed imports here with require below. I don't know anymore which is right but we should pick one ;)